src/iidm/Network.cpp

```diff
@@ -56,6 +70,15 @@ Network Network::readXml(std::istream& istream, const converter::ImportOptions&
     return converter::xml::NetworkXml::read(istream, options, anonymizer);
 }
 
+Network Network::readXml(std::istream& istream, const converter::Properties& parameters, const converter::Anonymizer& anonymizer) {
+    converter::ImportOptions options;
+    converter::Parameter THROW_EXCEPTION_IF_EXTENSION_NOT_FOUND_IMPORT_PARAMETER(THROW_EXCEPTION_IF_EXTENSION_NOT_FOUND_PARAMETER);
+    THROW_EXCEPTION_IF_EXTENSION_NOT_FOUND_IMPORT_PARAMETER.addAdditionalNames("throwExceptionIfExtensionNotFound");
```

Comment:
That's uggly: Why don't you add the additional names when you declare the parameter

Reply:
I agree... But the additional name is added in XMLImporter only (not in XMLExporter). 
I wanted to be Java strictly consistent ;)

Reply:
In java the static initializer is used to add the additional names during the construction of the constant. In your proposal your create a copy of the parameter each time readXml() is called:
```
    private static final Parameter THROW_EXCEPTION_IF_EXTENSION_NOT_FOUND_PARAMETER
            = new Parameter(THROW_EXCEPTION_IF_EXTENSION_NOT_FOUND, ParameterType.BOOLEAN, "Throw exception if extension not found", Boolean.FALSE)
            .addAdditionalNames("throwExceptionIfExtensionNotFound");
```